hcloud/instances.go

```diff
@@ -318,6 +356,7 @@ func (s hcloudServer) Metadata(networkID int64, cfg config.HCCMConfiguration) (*
 type robotServer struct {
 	*hrobotmodels.Server
 	robotClient robot.Client
+	recoder     record.EventRecorder
```

Comment:
```suggestion
	recorder     record.EventRecorder
```

Also needs to be fixed anywhere this field is used.